lib/ui/form/widget/form_survey_answer_multi_choice.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/model/question_model.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+
+class FormSurveyAnswerMultiChoice extends StatefulWidget {
+  final QuestionModel question;
+
+  const FormSurveyAnswerMultiChoice({super.key, required this.question});
+
+  @override
+  State<FormSurveyAnswerMultiChoice> createState() =>
+      _FormSurveyAnswerMultiChoiceState();
+}
+
+class _FormSurveyAnswerMultiChoiceState
+    extends State<FormSurveyAnswerMultiChoice> {
+  final List<int> _selectedIndexes = [];
+  late List<String> _options = [];
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  Widget _buildListItem(String title, bool isSelected) {
+    final selectedTextStyle = Theme.of(context).textTheme.labelMedium;
+    final notSelectedTextStyle = Theme.of(context)
+        .textTheme
+        .labelMedium!
+        .copyWith(fontWeight: FontWeight.w400);
+    final textStyle = isSelected ? selectedTextStyle : notSelectedTextStyle;
+    return SizedBox(
+      width: MediaQuery.of(context).size.width -
+          AppDimensions.answerMultiChoiceMarginLength,
+      height: AppDimensions.answerMultiChoiceItemHeight,
+      child: Row(
+        children: [
+          Expanded(
+            child: Text(
+              title,
+              style: textStyle,
+              maxLines: 1,
```

Comment:
Since we define `maxLines` here, should we also define `overflow` to truncate the text when it's longer than 1 line? 🤔